mesprojects.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
```

Comment:
Try to follow a standard on page names and try to use it in English. So, "my-projects.html"